content/design-systems-for-developers/react/en/build.md

```diff
@@ -101,129 +115,86 @@ The decorator will ensure the `GlobalStyle` is rendered no matter which story is
 
-Add the addon in `.storybook/main.js`:
+Let's see how they work, by adding a new story in the `Avatar` component, located in `src/Avatar.stories.js`:
 
 ```javascript
-//.storybook/main.js
-
-module.exports = {
-  stories: ['../src/**/*.stories.js'],
-  addons: [
-    '@storybook/preset-create-react-app',
-    '@storybook/addon-actions',
-    '@storybook/addon-links',
-    '@storybook/addon-storysource',
-    '@storybook/addon-knobs',
-  ],
-};
-```
-
-Add a story that uses knobs in `src/Avatar.stories.js`:
-
-```javascript
-//src/Avatar.stories.js
+// src/Avatar.stories.js
 
 import React from 'react';
-import { withKnobs, select, boolean } from '@storybook/addon-knobs';
 
 // …
 
-export const knobs = () => (
-  <Avatar
-    loading={boolean('Loading')}
-    size={select('Size', ['tiny', 'small', 'medium', 'large'])}
-    username="Dominic Nguyen"
-    src="https://avatars2.githubusercontent.com/u/263385"
-  />
-);
+// New story using controls
+const Template = args => <Avatar {...args} />;
 
-knobs.story = {
-  decorators: [withKnobs],
+export const Controls = Template.bind({});
+controls.args = {
+  loading: false,
+  size: 'tiny',
+  username: 'Dominic Nguyen',
+  src: 'https://avatars2.githubusercontent.com/u/263385',
 };
 ```
 
-Notice the Knobs tab in the addon panel. We instrumented the “Size” select element that allows us to cycle through the supported Avatar sizes `tiny`, `small`, `medium`, and `large`. You can instrument other props with knobs as well to create an interactive playground for the component.
+Notice the Controls tab in the addon panel. We instrumented the “size” select element that allows us to cycle through the supported Avatar sizes `tiny`, `small`, `medium`, and `large`. The same was applied to the remainder component's props (“loading”,“username” and “src”), allowing us to create a simple and user friendly way to interact and stress test components.
 
-![Storybook knobs addon](/design-systems-for-developers/storybook-addon-knobs.gif)
+<video autoPlay muted playsInline loop>
+  <source
+    src="/design-systems-for-developers/storybook-addon-controls-6-0.mp4"
+    type="video/mp4"
+  />
+</video>
 
-That said, knobs don’t replace stories. Knobs are great for exploring the edge cases of the components. Stories are used for showcasing the intended cases.
+That said, Controls don’t replace stories. They are great for exploring the edge cases of the components. Stories are used for showcasing the intended cases.
 
 We'll visit the Accessbility and Docs addons in later chapters.
```

Comment:
```suggestion
We'll visit the Accessibility and Docs addons in later chapters.
```

Reply:
Fixed with the latest commit. Marking as solved. Thanks for the feedback